application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -93,9 +93,38 @@
         #if($isChecked)X#end
       #else
         &lt;label&gt;
-          &lt;input type="#if($isRadio)radio#{else}checkbox#end"
-            #if($isCurrentUser)id="xpoll-proposal$proposal.hashCode()"#end name="$user" value="$proposal" $checked
-            $disabled /&gt;
+         ## condorcet method
+         #if ($pollObj.getValue('type') == '2')
+           #set ($indexOfProposal = $votes.indexOf($proposal) + 1)
+           #set ($noOption = '')
+           #if ($indexOfProposal &lt;  1)
+             #set ($noOption = 'selected')
+           #end
+
+           #if ($isCurrentUser &amp;&amp; $pollObj.getValue('status') == 'active')
+             &lt;select class="xpollSelect" proposal="$proposal"&gt;
```

Comment:
Is this `proposal` a custom attribute? Is the document still valid when using it? HTML 5 lets you use `data-*` and still having the document valid.

Reply:
Nope. The document was invalid. I changed it to "data-proposal".